### PR TITLE
Bump version to Mattermost 7.9.6

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Bump Mattermost Server version to pull the latest plugin api docs.

#### Ticket Link
None

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,10 +3,10 @@ module mattermost-developer-documentation
 go 1.18
 
 require (
-	// mmgoget: github.com/mattermost/mattermost-server/v6@v7.3.0 is replaced by -> github.com/mattermost/mattermost-server/v6@5d1fe271e6
-	github.com/mattermost/mattermost-server/v6 v6.0.0-20220914120725-5d1fe271e68c
+	// mmgoget: github.com/mattermost/mattermost-server/v6@v7.9.6 is replaced by -> github.com/mattermost/mattermost-server/v6@d0e672d94d
+	github.com/mattermost/mattermost-server/v6 v6.0.0-20230705082526-d0e672d94dee
```
Comment (mickmister) [inline go.mod:7]: Curious what the server dep is used for in the dev docs repo

**Review by mickmister — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (mickmister) [reply go.mod:7]: > to pull the latest plugin api docs.

Ah makes sense

**Review by spirosoik — APPROVED** [review]
I don't have much context though LGTM

**Comment by hanzei** [comment]
Replaced by https://github.com/mattermost/mattermost-developer-documentation/pull/1265

**Pull request closed without merging**